### PR TITLE
Offensive language discovered in the last name field (maxpreps.com)

### PR DESCRIPTION
**Pull request by nicolasdanelon**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -272,6 +272,12 @@ While not the dumbest password rule, still dumb.
 
 |Jitterbit|
 
+`Maxpreps <http://www.maxpreps.com/>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`Natalie Weiner <https://twitter.com/natalieweiner/status/1034533245839450113?s=19>`__
+ can't sign in because she's lastname is offensive language for the website
```
Comment (duffn) [inline README.rst]: Small grammatical error. This should be `because her last name`.

File: README.rst [hunk 2]
```diff
@@ -289,7 +295,7 @@ When creating a new account they enforce some password rules like: length must b
 between 8 and 15 characters and there must be one upper case, one lower case letter
 and one number.
 
-|MetLife|
+| |MetLife|
```
Comment (duffn) [inline README.rst]: Please revert this to the original `|MetLife|`.

**Comment by duffn** [comment]
Thanks for your contribution! Just two small comments.

File: README.rst (hunk 2 above)
Comment (nicolasdanelon) [reply README.rst]: :heavy_check_mark:

**Comment by duffn** [comment]
Thanks!

**Pull request merged**